deploy/olm-catalog/keepalived-operator/0.3.0/keepalived-operator.v0.3.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,303 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "redhatcop.redhat.io/v1alpha1",
+          "kind": "KeepalivedGroup",
+          "metadata": {
+            "name": "keepalivedgroup-workers"
+          },
+          "spec": {
+            "blacklistRouterIDs": [
+              1,
+              2
+            ],
+            "image": "registry.redhat.io/openshift4/ose-keepalived-ipfailover",
+            "interface": "ens3",
+            "nodeSelector": {
+              "node-role.kubernetes.io/loadbalancer": ""
+            }
+          }
+        }
+      ]
+    capabilities: Deep Insights
+    categories: Networking, Cloud Provider
+    certified: "false"
+    containerImage: quay.io/redhat-cop/keepalived-operator:v0.3.0
+    createdAt: 08/25/2020
+    description: This operator provides a simple way to create self-hosted load balancers
+      using keepalived.
+    repository: https://github.com/redhat-cop/keepalived-operator
+    support: Best Effort
+  name: keepalived-operator.v0.3.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: KeepalivedGroup is the Schema for the keepalivedgroups API
+      kind: KeepalivedGroup
+      name: keepalivedgroups.redhatcop.redhat.io
+      version: v1alpha1
+  description: "The objective of the keepalived operator provides is to allow for
```

Comment:
This description appears to be from the old README.md which has no longer working examples/old links to the OCP docs.

Reply:
done